onnxruntime/python/onnxruntime_pybind_state_common.h

```diff
@@ -169,6 +172,7 @@ extern std::string nuphar_settings;
 namespace onnxruntime {
 ProviderInfo_CUDA* TryGetProviderInfo_CUDA();
 ProviderInfo_CUDA& GetProviderInfo_CUDA();
+ProviderInfo_OpenVINO* GetProviderInfo_OpenVINO();
```

Comment:
Seems like some of the pipelines are failing:
![image](https://user-images.githubusercontent.com/25233514/141196948-cb60fef4-b49e-45ee-b7c9-071276328fe6.png)

Reply:
Fixed. Resolved